source/carbon/recommendations.rst

```diff
@@ -0,0 +1,21 @@
+Recommmendations
+----------------
+
+The plugin provides a diagnosis view for supported assets, in the tab Environmental impact. The administrator
+should refer to this page to easily find out which data is missing for a single asset.
+
+Mandatory informations are:
+
+*  A location with a country or a state filled
+*  A startup date or a delivery date
+*  A usage profile for computers
+*  a model or a type
+*  A power consumption in the model or the type of the asset
+*  Financial information enabled with one of the fllowing dates filled (ordered by precedence)
+  * use date
+  * delivery date
+  * order date
+
+Optional informations are:
+
+* Category of computer filled in each computer type
```

Comment:
we may tell that this value improves accuracy of data computed by Boavizta.

Also, depending on the category, some impact criterias will or will not be computed.